packages/design-system/src/components/HeaderBar/docs/HeaderBar.js

```diff
@@ -6,7 +6,7 @@ import Button from '../../Button';
 import Dropdown from '../../Dropdown';
 import Link from '../../Link';
 import { Icon } from '../../Icon';
-import Toggle from '../../Toggle';
+import { ButtonIconToggle } from '../../ButtonIcon';
```

Comment:
HeaderBar stories looks strange 
![image](https://user-images.githubusercontent.com/26482371/150836281-7e557630-8146-49d1-a302-98030c29b4d6.png)

Reply:
yep, it breaks the current styles. Is `HeaderBar` actually used though? If so then I'll have to create specific components for this.

Reply:
Alright, I'm leaving this as-is now for two reasons: 

- HeaderBar as a component is just a composition pattern. Not sure how I feel about this at this stage, since composition for external components isn't great. But regardless, this is an accurate story representing what can be done with the HeaderBar family of elements, it's not a standalone `<HeaderBar />`. 

- When (or if) we rework HeaderBar in CSS modules, we'll challenge that pattern. If we need to provided `HeaderBarToggle` and `HeaderBarLink` components we will happily to do.